### ISSUE TITLE
Show Keybind next to item

### ISSUE THREAD

**Issue by barnabasJ** [alexwforsythe/tmux-which-key#12]
One thing I really like about the neovim which-key plugin is that it uses the currently configured mappings. 
This is a good way to explore what actions you can take, but if there is something you do often, you can just remember the keybind and use that to do the same thing faster.

It would be nice if tmux-which-key could also show the key bind for an item if applicable. This way, it would also serve as a learning tool to help you remember the key binds to use for certain actions.

**Comment by ramarivera** [comment]
Hey @alexwforsythe 👋🏻 
Would you be open to a PR (trying, and possibly stumbling along the way) implementing this, if even possible?

**Comment by alexwforsythe** [comment]
Yes, I am open to this! In fact, I was just thinking about this when replying to #11.

Ideally, we would read key bindings from the user's root keymap and have some way to associate them with the menu items where applicable. It might look good to have the keybindings aligned to the right side of the popup menu and the menu items aligned to the left.
